internal/testrunner/runners/system/servicedeployer/kubernetes.go

```diff
@@ -132,15 +132,20 @@ func (ksd KubernetesServiceDeployer) installCustomDefinitions() error {
 
 var _ ServiceDeployer = new(KubernetesServiceDeployer)
 
-func findKubernetesDefinitions(definitionsDir string) ([]string, error) {
-	files, err := filepath.Glob(filepath.Join(definitionsDir, "*.yaml"))
-	if err != nil {
-		return nil, errors.Wrapf(err, "can't read definitions directory (path: %s)", definitionsDir)
+func containsKustomization(definitionsPath string) (bool, error) {
+	if _, err := os.Stat(definitionsPath); err != nil {
+		return false, errors.Wrapf(err, "can't read definitions directory (path: %s)", definitionsPath)
 	}
-
-	var definitionPaths []string
-	definitionPaths = append(definitionPaths, files...)
-	return definitionPaths, nil
+	// check if kustomization.yaml exists
+	if _, err := os.Stat(filepath.Join(definitionsPath, "kustomization.yaml")); err != nil {
```

Comment:
Nit. Please use a constant for `kustomization.yaml` string, specially if it is used in multiple places.